sbt-play-ebean/src/main/scala/play/ebean/sbt/SbtEbean.scala

```diff
@@ -108,7 +107,7 @@ object SbtEbean extends AutoPlugin {
     // code to load the config as it would be loaded in production
     def withClassLoader[T](block: ClassLoader => T): T = {
       val classpath = unmanagedResourceDirectories.value.map(_.toURI.toURL) ++ dependencyClasspath.value.map(_.data.toURI.toURL)
-      val classLoader = new URLClassLoader(classpath.toArray, null)
+      val classLoader = new URLClassLoader(classpath.toArray, this.getClass.getClassLoader)
```

Comment:
what was  missing in the previous classloader?  Looks like maybe the "dependencyClasspath" may be invalid if adding sbt's own classpath suddenly works....